commands/helpers.js

```diff
@@ -20,6 +20,16 @@ function requireParameterFormat (arg, formatFn, usageStr) {
   }
 }
 
+function memberFromQualifiedName (args, members) {
```

Comment:
code mostly lifted from stats.js; should probably migrate stats.js to using this form of helper.